components/List/BUILD

```diff
@@ -67,8 +67,8 @@ mdc_extension_objc_library(
     name = "Theming",
     deps = [
         ":List",
-        ":ColorThemer",
```

Comment:
Should the podspec be updated as well? Any docs?

Reply:
Will be done in #9190